polyfill/spatial-navigation-polyfill.js

```diff
@@ -211,9 +211,13 @@
     // When bestCandidate is found
     if (bestCandidate) {
       const container = bestCandidate.getSpatialNavigationContainer();
+      const elementStyle = window.getComputedStyle(container, null);
+      const overflowX = elementStyle.getPropertyValue('overflow-x');
+      const overflowY = elementStyle.getPropertyValue('overflow-y');
 
       // Scrolling container or document when the next focusing element isn't entirely visible
-      if (isScrollContainer(container) && !isEntirelyVisible(bestCandidate))
+      if (isScrollContainer(container) && !isEntirelyVisible(bestCandidate) && 
+          (overflowX !== 'hidden' || overflowY !== 'hidden'))
```

Comment:
The logic seems to be wrong.
What happens when {overflowX == visible (default), overflowY==hidden }
I assume It can be scrolled because overflowX !== 'hidden'
We need more TC to prevent such mistake.

Reply:
I think it won't be a problem because there is `isScrollContainer(container)` in this condition.
**_Overflowing container_** which has at least one axis of overflow is specified as 'scroll' or 'auto'.

So it wrote the condition "at least one axis of overflow is specified as 'scroll' or 'auto'." like 
"(overflowX !== 'hidden' || overflowY !== 'hidden')"

e.g.)
* {overflowX == scroll, overflowY==hidden }
* {overflowX == hidden, overflowY==scroll }
* {overflowX == auto, overflowY==hidden }
* {overflowX == hidden, overflowY==auto}
* {overflowX == scroll, overflowY==scroll}
* {overflowX == auto, overflowY==auto}
* {overflowX == scroll, overflowY==auto}
* {overflowX == auto, overflowY==scroll}

Reply:
I think.. 
Overflowing container which has at least one axis of overflow is specified as 'scroll' or 'auto' or **hidden**
Is overflow:hidden classified as container?

Reply:
Figuring out the element is overflowed or not doesn't count the value of `overflow`. It just sees `element.scrollHeight > element.clientHeight`.

Actually, I used wrong term of **_Overflowing container_** in here. 
Precisely, it is **_Overflowing container_ which can scroll by spatnav**.

And yes, `overflow:hidden` is already classified as a spatnav container.
You can see that in https://github.com/WICG/spatial-navigation/pull/198/files and `isScrollContainer()`.

Note that this change is about `isScrollable(element, dir)` (precisely, isScrollable by spatial navigation), not `isScrollableContainer`.

Reply:
When you check the actual operation,
internal/scrollable-container-test.html (fix-isScrollContainer branch)
the 5th overflow:hidden container is be scrolled. 
But I remember what we determine is overflow:hidden container should not be scrolled by spatNav.
>I assume It can be scrolled because overflowX !== 'hidden'

Reply:
I tried checking the operation with "--spatial-navigation-action': focus" and overflow:hidden.
but When I add the "'--spatial-navigation-action': focus" property, the snav is not working now.
once the whole modification of polyfill is done, please check the operation with "--spatial-navigation-action': focus"

Reply:
I simplify this condition via https://github.com/WICG/spatial-navigation/commit/08fc6beca596a02a558d0701d89e197627aa6da8.
I'll check the combined behavior with `spatial-navigation-action` while reviewing https://github.com/WICG/spatial-navigation/pull/199.